src/stories/AppMenu/AppMenuItem.stories.ts

```diff
@@ -182,7 +182,44 @@ export const Shortcut: Story = {
             <AAppMenuButton :open="open" v-bind="aria">Edit</AAppMenuButton>
           </template>
           <AAppMenuItem>Cut</AAppMenuItem>
-          <AAppMenuItem shortcut="Ctrl C">
+          <AAppMenuItem :shortcut="{modifiers: ['ctrl'], keySequence: 'C'}">
```

Comment:
I would use lower case c `<AAppMenuItem :shortcut="{modifiers: ['ctrl'], keySequence: 'c'}">`

it's capitalized by the `KeyboardShortcut` component